src/base/bittorrent/session.cpp

```diff
@@ -440,6 +443,11 @@ Session::Session(QObject *parent)
     // initialize PortForwarder instance
     Net::PortForwarder::initInstance(m_nativeSession);
 
+#if LIBTORRENT_VERSION_NUM >= 10100
+    initMetrics();
+    m_statsUpdateTime = QDateTime::currentDateTime();
```

Comment:
So what do you prefer? Use `QDateTime::currentDateTimeUTC()` or yet another timer?